.scripts/list-of-samples.json

```diff
@@ -27,6 +29,7 @@
     "production",
     "protobufs",
     "query-subscriptions",
+    "replay-history",
```

Comment:
this and above are old—you can rm here and rm the directories locally

Reply:
Thank you